packages/core/src/components/modal/modal.tsx

```diff
@@ -68,11 +67,6 @@ export class AtomModal {
   }
 
   componentDidLoad() {
-    /* @todo it's needed to prevent a ionic error. In the version 8.0 it was fixed, remove it after the upgrade.
```

Comment:
BACKDROP_NO_SCROLL is used in more places on this file, guess it can be done after migration in a dedicated task. What do you think?

Reply:
perfect! I [return it]([7ab6f48](https://github.com/juntossomosmais/atomium/pull/684/commits/7ab6f487bbb98d2ca5592eaa7d5f4f8d7b9edc8e)). could u check it with @jowjow22 and @matheus-rosa-jsm next week?

Reply:
I just [commented](https://github.com/juntossomosmais/atomium/issues/535#issuecomment-2578480361)  on the issue to remember us. ;)